tests/test_modeling_longformer.py

```diff
@@ -194,6 +195,23 @@ def create_and_check_longformer_for_question_answering(
         self.parent.assertListEqual(list(result["end_logits"].size()), [self.batch_size, self.seq_length])
         self.check_loss_output(result)
 
+    def create_and_check_longformer_for_sequence_classification(
```

Comment:
Awesome that for including the tests!

Reply:
learned this from our previous discussion :)